CHANGES.txt

```diff
@@ -81,6 +81,22 @@ Changes
 Fixes
 =====
 
+- Remove horizontal scroll from the console editor in the Admin UI.
+
+- Show all nodes in the Admin UI shard view, even those which have no shards.
+
+- Fixed an issue that cause the status-bar of the Admin UI to show an error if
+  the user has an add blocker enabled.
+
+- Fixed an issue that caused extra spaces to be added to the formatted console
+  results in the Admin UI.
+
+- Fixed an issue that caused ``NULL`` generated column values not to be
+  displayed in the tables view in the Admin UI.
+
+- Show messages when the table list is empty in the Tables view of the
```

Comment:
Fixed Tables view of the Admin UI to show messages when the table list is empty.